packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -48,11 +49,28 @@ export function disableCLITelemetry() {
   );
 }
 
+async function getConfigAggregator(): Promise<ConfigAggregator> {
+  const origCurrentWorkingDirectory = process.cwd();
+  const rootWorkspacePath = getRootWorkspacePath();
+  // Change the current working directory to the project path,
+  // so that ConfigAggregator reads the local project values
+  process.chdir(rootWorkspacePath);
+  const configAggregator = await ConfigAggregator.create();
+  // Change the current working directory back to what it was
+  // before returning
+  process.chdir(origCurrentWorkingDirectory);
+  return configAggregator;
+}
+
 export async function isCLITelemetryAllowed(): Promise<boolean> {
   try {
-    const disabledConfig =
-      (await ConfigUtil.getConfigValue(SFDX_CONFIG_DISABLE_TELEMETRY)) || '';
-    return disabledConfig !== 'true';
+    const configAggregator = await getConfigAggregator();
+    const disableTelemetry:
+      | string
+      | undefined = configAggregator.getPropertyValue(
```

Comment:
what's the extra `|`

Reply:
🤷🏽‍♂️  When I save it comes back

https://user-images.githubusercontent.com/46458081/182253935-02df422e-fcbc-4a12-a019-223d4240b1aa.mp4

: